authserver/api/client.py

```diff
@@ -114,6 +115,8 @@ def post(self, action, id: str = None):
                         except Exception as e:
                             client_metadata[k] = v
             client.set_client_metadata(client_metadata)
+            client.client_id_issued_at = int(time.time())
```

Comment:
Also curious why you used the `time` module here as opposed to `datetime` (i.e. datetime.timestamp(datetime.now()))

Reply:
I created the timestamp the same way the deprecated version of OAuth2ClientMixin did: https://github.com/lepture/authlib/blob/64390bfa02f14923e2dd1150d2775f1782bcf166/authlib/flask/oauth2/sqla.py#L22-L24

Then, we’ll have consistency among new and previously created clients.